### ISSUE TITLE
Getting silently disconneceted after SIDCC cookie implementation

### ISSUE THREAD

**Issue by larstiq** [rogora/hangish#53]
Running version 0.9.2 from the Jolla store I notice that at some point I no longer receive messages from people (that I do see in, say, https://hangouts.google.com).  If I then try to send a message myself it fails and shows "Error, msg not sent!".

Workaround is to close the app and start it again, so I suppose it has something to do with refreshing auth.

**Comment by rogora** [comment]
Yes, I know... there is something broken with the current handling of the authentication, I'd like to find time to work on this...
